src/plugins/dna_stat/src/DNAStatMSAProfileDialog.cpp

```diff
@@ -123,6 +124,14 @@ void DNAStatMSAProfileDialog::accept() {
     QDialog::accept();
 }
 
+void DNAStatMSAProfileDialog::alignmentIsTooBig() {
+    warningLabel->setText(tr("<b><font color=%1>%2</font><br></br></b>")
+        .arg(Theme::errorColorLabelHtmlStr())
+        .arg(tr("Warning: Alignment is too big. Generated grid profile couldn't be shown properly or even will crash application.<br>You can open generated profile by your own.")));
```

Comment:
Change the long text to a simple "Warning: report is too big to be shown in UGENE"